.github/workflows/package_warnings_as_errors.yml

```diff
@@ -0,0 +1,26 @@
+name: Test-suite with Python warnings as errors
+on:
+  # Scheduled trigger every Sunday at 2:35am UTC
+  schedule:
+  - cron: '35 2 * * 0'
+  workflow_dispatch:
+  pull_request:
+
+concurrency:
+  group: warnings-as-errors-tests-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test-warnings-as-errors:
+    uses: ./.github/workflows/interface-unit-tests.yml
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+    secrets:
+      codecov_token: ${{ secrets.CODECOV_TOKEN }}
+    with:
+      branch: ${{ github.ref }}
+      skip_ci_test_jobs: 'qcut-tests'
+      python_warning_level: 'error'
+      pipeline_mode: 'warnings'
```

Comment:
Is this a valid pipeline_mode!?

Reply:
I recently added this to support the above conditional separations of the tests. I don't see a better way to achieve this.